src/Umbraco.Web.Common/ActionsResults/ValidationErrorResult.cs

```diff
@@ -1,10 +1,19 @@
-﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Umbraco.Cms.Core.Models;
 using Umbraco.Cms.Core.Models.ContentEditing;
 using Umbraco.Extensions;
 
 namespace Umbraco.Cms.Web.Common.ActionsResults
 {
+    // TODO: This should probably follow the same conventions as in aspnet core and use ProblemDetails
+    // and ProblemDetails factory. See https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Core/src/ControllerBase.cs#L1977
+    // ProblemDetails is explicitly checked for in the application model.
+    // In our base class UmbracoAuthorizedApiController the logic is there to create a ProblemDetails.
+    // However, to do this will require changing how angular deals with errors since the response will
+    // probably be different. Would just be better to follow the aspnet patterns.
```

Comment:
Agreed, but I think it is better to wait for the new Backoffice, before we make such change.